build.sbt

```
@@ -20,6 +20,16 @@ lazy val `docs` = project
   .settings(skip in publish := true)
   .dependsOn(allProjects.map(ClasspathDependency(_, None)): _*)
 
+lazy val `website` = project
+  .in(file("website-docs"))
+  .aggregate(allProjects: _*)
+  .enablePlugins(MdocPlugin)
+  .enablePlugins(MicrositesPlugin)
+  .settings(skip in publish := true)
+  .settings(mdocIn := file("website-docs/docs"))
+  .settings(micrositeSettings)
+  .dependsOn(allProjects.map(ClasspathDependency(_, None)): _*)
```

You don't need to use a different folder, since we are already setting a different one for `mdocIn`:

```suggestion
lazy val `website` = project
  .aggregate(allProjects: _*)
  .enablePlugins(MdocPlugin)
  .enablePlugins(MicrositesPlugin)
  .settings(skip in publish := true)
  .settings(mdocIn := file("website/docs"))
  .settings(micrositeSettings)
  .dependsOn(allProjects.map(ClasspathDependency(_, None)): _*)
```